www/json/trip_confirm_options.json.sample

```diff
@@ -41,12 +41,12 @@
        {"text":"Transit transfer", "value":"transit_transfer"},
        {"text":"Shopping","value":"shopping"},
        {"text":"Meal","value":"meal"},
-       {"text":"Pick-up/Drop off 👤","value":"pick_drop_person"},
-       {"text":"Pick-up/Drop off 📦","value":"pick_drop_item"},
-       {"text":"Personal/Medical","value":"personal_med"},
+       {"text":"Pick-up/ Drop off Person","value":"pick_drop_person"},
+       {"text":"Pick-up/ Drop off Item","value":"pick_drop_item"},
+       {"text":"Personal/ Medical","value":"personal_med"},
        {"text":"Access Recreation","value":"access_recreation"},
-       {"text":"Recreation/Exercise","value":"exercise"},
-       {"text":"Entertainment/Social","value":"entertainment"},
+       {"text":"Recreation/ Exercise","value":"exercise"},
+       {"text":"Entertainment/ Social","value":"entertainment"},
```

Comment:
Ditto

Reply:
I addressed this in my recent PR for e-mission-translate: https://github.com/e-mission/e-mission-translate/pull/19